### PR TITLE
 Add <ScrollToTopOnMount /> to container and pod pages

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes https://jira.coreos.com/browse/CONSOLE-593

After:
qpahl5qnrb

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
     "react-modal": "3.x",
     "react-redux": "5.x",
     "react-router-dom": "4.x",
+    "react-router-scroll-4": "^1.0.0-beta.1",
```
Comment (alecmerdler) [inline frontend/package.json]: We should not be relying on beta software.

Comment (benjaminapetersen) [inline frontend/package.json]: So this one is a fork: 
- fork: https://github.com/ytase/react-router-scroll  (this)
- orig: https://github.com/taion/react-router-scroll

But original doesn't support the behavior.  Looking it over.

**Comment by spadgett** [comment]
@alecmerdler @benjaminapetersen I was looking at this with @rhamilto on other solutions. I don't think we need the library.

**Comment by rhamilto** [comment]
Take 2 using [ScrollToTopOnMount](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/guides/scroll-restoration.md#scroll-to-top).  With @spadgett's help, we were able to determine the fix is necessary between pod and container pages because the resource doesn't change.  We'll need to employ this same fix for any time we have routes that change but resources that do not.

**Comment by spadgett** [comment]
LGTM, but probably worth a comment explaining

**Comment by rhamilto** [comment]
Rebased and added comment explaining use for `<ScrollToTopOnMount />`

**Comment by spadgett** [comment]
Looks like you have a linter error:

```
[4m/go/src/github.com/openshift/console/frontend/public/components/container.jsx[24m
  [2m6:118[22m  [31merror[39m  Multiple spaces found before '}'  [2mno-multi-spaces[22m
```

**Comment by rhamilto** [comment]
Linter error fixed.

**Pull request merged**